src/app/tvshows/store/tvshows.selectors.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@angular/core';
+import { select } from '@angular-redux/store';
+import { Observable } from 'rxjs/Observable';
+import { TvShowData } from './tvshows.models';
+import { RootState } from '../../store';
+
+export class TvShowSelectors {
+  @select(getSortedResults) public readonly tvshows$: Observable<TvShowData[]>;
+  @select(getIsFetching) public readonly isFetching$: Observable<boolean>;
+}
+
+/*
+ I have to define these as functions and then pass them to @select() due to a
+ typescript bug which causes builds to fail when -aot is enabled.
+
+ I originally had these specified as arrow functions within the @selector()
+ decorator
+ */
+
+function getSortedResults(state: RootState) {
+  return state.tvshows.searchResults.sort(sortShowsByTitle);
+}
+
+function getIsFetching(state: RootState) {
+  return state.tvshows.isFetching;
+}
+
+function sortShowsByTitle(a: TvShowData, b: TvShowData) {
```

Comment:
This function name is confusing. It's not actually sorting, but its more like comparator function. I'd call it `showsSortComparer` or something. This is a prime candidate for using FP style (possibly using Ramda instead or to simplify the logic).

Reply:
Will do.